typescript/introduction/intro-typescript/typescript-dependencies.md

```diff
@@ -0,0 +1,78 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Get started with TypeScript
+
+---
+## Content
+
+Installing TypeScript is as easy as writing a line of code. Using Node's package manager, all you have to type is:
```

Comment:
I'd drop `Node's package manager` and use `npm` directly

Reply:
👍 since there are so many expansions https://github.com/npm/npm-expansions

Reply:
Yeah, good point 👍

Reply:
```suggestion
Installing TypeScript is as easy as writing a line of code. Using NPM, all you have to type is:
```